keras_cv/bounding_box/utils_test.py

```diff
@@ -38,3 +38,34 @@ def test_clip_to_image(self):
         )
         self.assertAllGreaterEqual(bboxes_out, 0)
         self.assertAllLessEqual(bboxes_out, 1)
+
+    def test_pad_with_sentinels(self):
+        bounding_boxes = tf.ragged.constant(
+            [[[1, 2, 3, 4, 5], [1, 2, 3, 4, 5]], [[1, 2, 3, 4, 5]]]
+        )
+        padded_bounding_boxes = bounding_box.pad_with_sentinels(bounding_boxes)
+        expected_output = tf.constant(
+            [
+                [[1, 2, 3, 4, 5], [1, 2, 3, 4, 5]],
+                [[1, 2, 3, 4, 5], [-1, -1, -1, -1, -1]],
+            ]
+        )
+        self.assertAllEqual(padded_bounding_boxes, expected_output)
+
+    def test_filter_sentinels(self):
+        bounding_boxes = tf.ragged.constant(
+            [[[1, 2, 3, 4, 5], [1, 2, 3, 4, -1]], [[1, 2, 3, 4, 5]]]
+        )
+        filtered_bounding_boxes = bounding_box.filter_sentinels(bounding_boxes)
+        expected_output = tf.ragged.constant([[[1, 2, 3, 4, 5]], [[1, 2, 3, 4, 5]]])
+        self.assertAllEqual(filtered_bounding_boxes, expected_output)
+
+    def test_pad_with_class_id(self):
```

Comment:
lets add a unit test covering the case where `class_id` is already present

Reply:
(the new unit test should error out)

Reply:
added